projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+
+* ❌ date is not known in advance
+* ❌ location is not known in advance
+
+In order to create a bookable school event you must use
+
+* calendarType `permanent`
+* the URL of the dummy place as `location.@id` in the `POST /events` request of the bookable events
+
+**URL of the dummy place for bookable events**:
+
+* Test environment: `https://io-test.uitdatabank.be/place/3b92c85b-a923-4895-85f5-ed056dae11e2`
+* Production environment: `https://io.uitdatabank.be/place/c3f9278e-228b-4199-8f9a-b9716a17e58f`
+
+## Creating school events
+
+For the creation of school events several extra requirements apply:
+
+1. The `audienceType` must be set to `education`
+2. The event must have an `organizer` that has the `Cultuurkuur` label
+3. Specific education related `labels` are mandatory
+4. In case of a [guided tour](#guided-tours) or a [bookable event](#bookable-events): the `calendarType` must be set to `permanent`
+5. In case of a [bookable event](#bookable-events): the `dummy place for bookable events` must be used for the location
+
+### audienceType
+
+For school events you must include an extra property `audienceType` and set the value for the property to `education`.
+
+```js
+{
+  "audienceType": "education"
+}
+```
+
+### organizer
+
+Every school event must be linked to an [existing organizer page on Cultuurkuur](https://www.cultuurkuur.be/organisaties).
+
+**How to check if your organization already has an organizer page on Cultuurkuur:**
+
+1. Go to <https://www.cultuurkuur.be/organisaties>
+2. Type in the name of your organization in the search box, e.g. `hetpaleis`
+3. If a match is found, open the detail page of the organization
+4. In the url of the browser you'll find the identifier of the organizer in UiTdatabank:
+
+![](../../assets/images/cultuurkuur-search-organizer.png)
+
+Prefix this value with the host url of the according environment and use this as the value for the `organizer.@id` property in the `POST /events` request:
+
+```js
+{
+  "organizer": {
+    "@id": "https://io.uitdatabank.be/organizers/d319d57f-7400-4c16-aa19-8f04992da3fa"
+  }
+}
+```
+
+> Only in the case the organizer of your event does not already have its own page on Cultuurkuur yet, you can [create a new organizer on Cultuurkuur](https://www.cultuurkuur.be/faq/hoe-voeg-ik-mijn-organisatie-toe-op-cultuurkuur).
+
+### labels
+
+For school events, specific Cultuurkuur-related labels are mandatory. There are 3 different types of Cultuurkuur-related labels:
+
+1. **Target group labels** are used to specify if the if event is for students or for teachers.
+2. **Subject labels** are used to define the learning objective of the school event.
+3. **Education level labels**
+
+#### Target group labels
+
+Target group labels are used to specify if the if event is for students or for teachers.
+
+> A school event must have **exactly 1** target group label.
```

Comment:
I think that in English prose, the general rule is to spell out numbers smaller than 10 (see e.g. https://www.scribendi.com/academy/articles/when_to_spell_out_numbers_in_writing.en.html)

```suggestion
> A school event must have **exactly one** target group label.
```

Also applies to other info blocks below (e.g. "A school event must have **at least 1** subject label.")